Perl/README.md

```diff
@@ -0,0 +1,70 @@
+# Perl
+
+macOS ships with [Perl](https://www.perl.org), but it's not safe to
+rely on it for more than basic one-liners. Apple has broken it twice,
+and let it go very stale at least once. Many packages distributed with
+Homebrew depend on the version of Perl it installs, but for complete
+control of what version you're running and what packages are
+installed, there's also [Perlbrew](https://perlbrew.pl).
+
+## Installation
+
+### Homebrew
+
+```bash
+$ brew install perl cpanminus
```

Comment:
A small explanation to why `cpanminus` is needed would be nice.

Reply:
Added brief explanation.